lib/msf/core/framework.rb

```diff
@@ -82,6 +82,7 @@ def initialize(options={})
     self.plugins   = PluginManager.new(self)
     self.browser_profiles = Hash.new
     self.features = FeatureManager.instance
+    self.uuid_db   = Rex::JSONHashFile.new(::File.join(Msf::Config.config_directory, "payloads.json"))
```

Comment:
We intentionally removed this JSON file when we switched payload UUID tracking to the database. The proper fix to this issue would be to warn users without a database connected that payload UUID tracking will not work when the payload tracking options are set.